lib/katello/repository_types/yum.rb

```diff
@@ -0,0 +1,4 @@
+
+Katello::RepositoryTypeManager.register(::Katello::Repository::YUM_TYPE) do
+  description "Yum Repository types"
```

Comment:
I wonder if description is actually needed. I added more to make sense to me as a plugin user. I guess we could just make it a comment.